includes/admin/admin-actions.php

```diff
@@ -570,7 +570,11 @@ function give_validate_user_profile( $errors, $update, $user ) {
 		if( $donor ) {
 			// If Donor is attached with User, then validate first name.
 			if ( empty( $_POST['first_name'] ) ) {
-				$errors->add( 'empty_first_name', '<strong>ERROR:</strong>' . __( 'Please enter your first name.', 'give' ) );
+				$errors->add(
+					'empty_first_name',
+					sprintf( '%1$s %2$s', esc_html__( '<strong>ERROR:</strong>', 'give' ), __( 'Please enter your first name.', 'give' )
```

Comment:
@mehul0810 please separate HTML and translatable strings
```
sprintf(
    '<strong>%1$s:</strong> %2$s',
    __( 'ERROR', 'give' ),
    __( 'Please enter your first name.', 'give' )
);
```

Reply:
@ravinderk Improved Code